content/yaml-code-signing/code-signing-identities.md

```diff
@@ -0,0 +1,159 @@
+
+{{<notebox>}}
+Fetching certificates that Codemagic has not created is disabled because each certificate has a unique private key to which Codemagic has no access, thus rendering the certificate unusable.
+{{</notebox>}}
+
+The **Reference name**, certificate type, team, and expiration date are displayed for each added certificate.
+
+### iOS profiles
+
+You can upload provisioning profiles with the `.mobileprovision` extension. 
+
+Alternatively, you can automatically fetch the provisioning profiles from the Apple Developer Portal based on your team's App Store Connect API key. The bundle identifier is listed for every available profile along with it's name. The profiles are displayed by category: `Development profiles`, `Ad Hoc profiles`, `App Store profiles`, and `Enterprise profiles`. For each selected profile, it is necessary to provide a unique **Reference name**, which can be later used in `codemagic.yaml` to fetch the profile.
+
+The profile's type, team, bundle id, and expiration date are displayed for each profile added to Code signing identities. Furthermore, Codemagic will let you know whether a matching certificate exists under Code signing identities certificates in the `Certificate` field.
+
+### Android keystores
+
+Codemagic lets you upload your Android keystores. When uploading a keystore, it is required that the keystore password, key alias, and key password (if exists) are provided.
+
+Furthermore, it is required that a unique **Reference name** is assigned to each uploaded keystore, which can be used to fetch the keystore during the build using the `codemagic.yaml` configuration file. 
+
+For each keystore, its common name, issuer, and expiration date are displayed.
+
+
+Note that the uploaded keystore cannot be downloaded from Codemagic. It is crucial that you store a copy of the keystore file used in Codemagic as all subsequent builds released to Google Play should be signed with the same keystore.  
+
+## Referencing files in codemagic.yaml
+
+After uploading code signing files to Codemagic, these files can be fetched and used during the build by providing the reference names in the `codemagic.yaml` configuration.
+
+### Apple certificates and profiles
+
+Codemagic provides two means of fetching the required certificates and provisioning profiles during the build with the use of `codemagic.yaml`. Fetching can either be configured by specifying the distribution type and bundle identifier, or for more advanced use-cases, individual files can be fetched by their reference names.
+
+#### Fetching files by distribution type and bundle identifier
+
+To fetch all uploaded signing files matching a specific distribution type and bundle identifier during the build, define the `distribution_type` and `bundle_identifier` fields in your `codemagic.yaml` configuration. Note that it is necessary to configure both of the fields.
```

Comment:
I think it is important to note that when using `distribution_type` and `bundle_identifier` then specifing `profiles` and `certificates` is not allowed.